docs/cli.rst

```diff
@@ -73,6 +74,15 @@ If you need to quote the values in your YAML file, the explicit ``!env_var`` res
     # foobar.yaml
     AMQP_URI: !env_var "pyamqp://${RABBITMQ_USER:guest}:${RABBITMQ_PASSWORD:password}@${RABBITMQ_HOST:localhost}"
 
+You can provide many level of generic default value
+
+.. code-block:: yaml
+
+    # foobar.yaml
+    AMQP_URI: ${AMQP_URI:pyamqp://${RABBITMQ_USER:guest}:${RABBITMQ_PASSWORD:password}@${RABBITMQ_HOST:localhost}}
+
+this config accept AMQP_URI as environment variable, in this case RABBITMQ_* variables will not be used.
```

Comment:
`this config accepts AMQP_URI as an environment variable, if provided RABBITMQ_* nested variables will not be used.`